transfer/src/main/java/org/kinecosystem/transfer/receiver/presenter/AccountInfoError.kt

```diff
@@ -0,0 +1,15 @@
+package org.kinecosystem.transfer.receiver.presenter
+
+class AccountInfoError(val errorType: Type, val listener: IErrorActionClickListener? = null) {
```

Comment:
The IErrorActionClickListener does not belong here.
It is a listener for the dialog actions ....